src/health_checker.cc

```diff
@@ -20,6 +20,7 @@
 #include <boost/algorithm/string/join.hpp>
 #include <vector>
 #include <fstream>
+#include <stdlib.h>
```

Comment:
`<cstdlib>` (see [this reference](http://en.cppreference.com/w/cpp/utility/program/exit)).

Reply:
changed